bundler/lib/bundler/cli/fund.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Bundler
+  class CLI::Fund
+    attr_reader :options
+
+    def initialize(options)
+      @options = options
+    end
+
+    def run
+      Bundler.definition.validate_runtime!
+
+      groups = Array(options[:group]).map(&:to_sym)
+
+      deps = if groups.any?
+        Bundler.definition.dependencies_for(groups)
```

Comment:
Tests failed because I suggested this method which is also private, I guess it's fine to make it public too.

Reply:
I think so too; `specs_for` is already also public.